couchexport/writers.py

```diff
@@ -221,6 +221,18 @@ def _init_table(self, table_index, table_title):
         self.tables[table_index] = sheet
         self.table_indices[table_index] = 0
 
+    def get_stripped_data(self, row):
+        """
+        A generator that returns data stripped of HTML tags
+
+        .. NOTE:: This does not convert or strip HTML entities like `&amp;`
+
+        """
+        tag = re.compile('<[^<]+?>')
```

Comment:
I don't think this is the right place for this change, and I think this is likely to introduce more issues than it helps.

Our reports have an `export_rows` function (https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/generic.py#L880) that lets you return the rows specifically in the context of an export so that you can do things like unstyle them. that feels like a much more appropriate place to make a surgical fix rather than something that will affect anything that uses couchexport.

I'm going to propose reverting this change.